Json.More/JsonArrayTupleConverter.cs

```diff
@@ -22,10 +23,12 @@ public override bool CanConvert(Type typeToConvert)
 	/// <param name="typeToConvert">The type handled by the converter.</param>
 	/// <param name="options">The serialization options to use.</param>
 	/// <returns>A converter which is compatible with <paramref name="typeToConvert" />.</returns>
+	[UnconditionalSuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code", Justification = "<Pending>")]
+	[UnconditionalSuppressMessage("AOT", "IL3050:Calling members annotated with 'RequiresDynamicCodeAttribute' may break functionality when AOT compiling.", Justification = "<Pending>")]
```

Comment:
These should  be changed to RequiresDynamicCode/RequiresUnreferencedCode so that we can let callers know this won't work.

Reply:
I tried that.  I get a warning that says the attributes have to match those on the base method.  Since the base method doesn't have those attributes, it doesn't like me adding them.  But it's okay with these.

No one's ever going to be calling the `.CreateConveter()` method direction anyway, so they'll never see the warnings if I put those attributes here.

Reply:
Fair enough.

Reply:
It should be possible to work around this issue by annotating with RUC/RDC at the type level, for example:

https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/UnsupportedTypeConverterFactory.cs

Reply:
Nice. I didn't know you could annotation the type.  I'll add this.